templates/explore/repo_list.tmpl

```diff
@@ -7,13 +7,26 @@
 				{{end}}
 				<a class="name" href="{{.Link}}">
 					{{if or $.PageIsExplore $.PageIsProfileStarList }}{{if .Owner}}{{.Owner.Name}} / {{end}}{{end}}{{.Name}}
-					{{if .IsArchived}}<i class="archive icon archived-icon"></i>{{end}}
 				</a>
-				{{if .IsPrivate}}
-					<span class="middle text gold">{{svg "octicon-lock" 16}}</span>
-				{{else if and (not .IsMirror) (not .IsFork) (.Owner.Visibility.IsPrivate) }}
-					<span class="text gold">{{svg "octicon-internal-repo" 16}}</span>
-				{{else if .IsFork}}
+				{{if .IsArchived}}<span class="ui compact label">{{$.i18n.Tr "repo.desc.archived"}}</span>{{end}}
+				{{if .IsTemplate}}
+					{{if .IsPrivate}}
+						<span class="ui basic label">{{$.i18n.Tr "repo.desc.private_template"}}</span>
```

Comment:
Why not render 2 labels? E.g. `private` `template`?

Reply:
Takes more space and looks weird imho, makes unnecessary distinction with a visible margin and usually user is interested in content of both, not separately.